docs/development/setup-advanced.md

```diff
@@ -12,6 +12,14 @@ Start by [cloning your fork of the zulip repository][zulip-rtd-git-cloning]:
 
 `git clone --config pull.rebase https://github.com/YOURUSERNAME/zulip.git`
 
+As well as [connecting your fork to Zulip upstream with]
+(..git/cloning.html#step-1c-connect-your-fork-to-zulip-upstream)
+
+```
+git remote add upstream https://github.com/zulip/zulip.git
+git fetch upstream
```

Comment:
I think it'd be best to actually combine these commands with the above in all of these blocks, so that it's like this:
```
git clone --config pull.rebase https://github.com/YOURUSERNAME/zulip.git
git remote add upstream https://github.com/zulip/zulip.git
git fetch upstream
```

That way, users don't need to copy-paste multiple blocks in order to get the Git instructions.

And then we can just adjust the text for the linked zulip-rtd-git-cloning page to add a bit of explanation as to what this does.